src/community/advanced-data-card.tsx

```diff
@@ -250,7 +251,8 @@ type AdvancedDataCardProps = MaybeTouchableCard<{
     description?: string;
     descriptionLinesMax?: number;
     extra?: ReadonlyArray<RendersNullableElement<AllowedExtra>>;
-    extraDividerPadding?: 8 | 24;
+    extraDividerPadding?: 8 | 16 | 24;
+    noDivider?: boolean;
```

Comment:
As you already have `extraDividerPadding`, I'd call this prop `noExtraDivider`, to follow the same naming

Reply:
Okay